### PR TITLE
[const] Centralize constants for overrides

### PR DESCRIPTION
**Pull request by gregnazario**
### Description

This finally stops from slapping these long allowlists in multiple places

### CODE REVIEW

File: src/components/Table/VerifiedCell.tsx [hunk 1]
```diff
@@ -57,46 +66,6 @@ export function isBannedType(level: VerifiedType): boolean {
...
-const labsVerifiedTokens: Record<string, string> = {
-  "0x357b0b74bc833e95a115ad22604854d6b0fca151cecd94111770e5d6ffc9dc2b": "USDt",
-  "0xbae207659db88bea0cbead6da0ed00aac12edcdda169e591cd41c94180b46f3b": "USDC",
-  "0xd39fcd33aedfd436a1bbb576a48d5c7c0ac317c9a9bb7d53ae9ffb41e8cb9fd9":
-    "Find Out Points",
-  "0x50038be55be5b964cfa32cf128b5cf05f123959f286b4cc02b86cafd48945f89": "BUIDL",
-};
-const MARKED_AS_SCAM = "Marked as scam";
-const MARKED_AS_POSSIBLE_SCAM = "Marked as possible scam";
-const labsBannedTokens: Record<string, string> = {
```
Comment (bv-aptos) [inline src/components/Table/VerifiedCell.tsx:77]: Our current banned list includes:

"0xf658475dc67a4d48295dbcea6de1dc3c9af64c1c80d4161284df369be941dafb::moon_coin::MoonCoin",
      "0xbbc4a9af0e7fa8885bda5db08028e7b882f2c2bba1e0fedbad1d8316f73f8b2f::Questss::Quests",
      "0xbbc4a9af0e7fa8885bda5db08028e7b882f2c2bba1e0fedbad1d8316f73f8b2f::quest::Quest",
      "0xbc106d0fef7e5ce159423a1a9312e011bca7fb57f961146a2f88003a779b25c2::QUEST::QUEST",
      "0xbbc4a9af0e7fa8885bda5db08028e7b882f2c2bba1e0fedbad1d8316f73f8b2f::Questseess::Questsee",
      "0xbbc4a9af0e7fa8885bda5db08028e7b882f2c2bba1e0fedbad1d8316f73f8b2f::graffio::GRAFFIO",
      "0xbbc4a9af0e7fa8885bda5db08028e7b882f2c2bba1e0fedbad1d8316f73f8b2f::ograffioo::Ograffioo",
      "0xbbc4a9af0e7fa8885bda5db08028e7b882f2c2bba1e0fedbad1d8316f73f8b2f::ograffio::Ograffio",
      "0xbbc4a9af0e7fa8885bda5db08028e7b882f2c2bba1e0fedbad1d8316f73f8b2f::memes::Memecoins",
      "0xbbc4a9af0e7fa8885bda5db08028e7b882f2c2bba1e0fedbad1d8316f73f8b2f::meme::Memecoin",
      "0xbbc4a9af0e7fa8885bda5db08028e7b882f2c2bba1e0fedbad1d8316f73f8b2f::tortugafin::Tortugafin",
      "0x397071c01929cc6672a17f130bd62b1bce224309029837ce4f18214cc83ce2a7::USDC::USDC",
      "0xbbc4a9af0e7fa8885bda5db08028e7b882f2c2bba1e0fedbad1d8316f73f8b2f::Ogrf::Ogrff",
      "0x24a808cbdeee2e8b771fec7ce9140bab2cbbf89426bf135d6b5d88e6736bb038::EZMoney::EZ_Money",
      "0x48327a479bf5c5d2e36d5e9846362cff2d99e0e27ff92859fc247893fded3fbd::APTOS::APTOS",
      "0xbbc4a9af0e7fa8885bda5db08028e7b882f2c2bba1e0fedbad1d8316f73f8b2f::Memecoins::Aptmeme",
      "0xbde343d6d2a7b407b1314c6c6ef771ebd72b95f9f67b688bbaea5ece4b98bb57::maincoin::Aptmeme",
      "0xbe5e8fa9dd45e010cadba1992409a0fc488ca81f386d636ba38d12641ef91136::maincoin::Aptmeme",
      "0xe1d39a72bd69bc2ebfe008bb925badb23a32883b077218b9e167f74cf703db1a::uptos::UptosCoin",
      "0x50788befc1107c0cc4473848a92e5c783c635866ce3c98de71d2eeb7d2a34f85::aptos_coin::AptosCoin",
      "0x50788befc1107c0cc4473848a92e5c783c635866ce3c98de71d2eeb7d2a34f85::apt_rewards::APTRewards",
      "0x50788befc1107c0cc4473848a92e5c783c635866ce3c98de71d2eeb7d2a34f85::aptos_coin::AptosCoin",
      "0xb8aed93b62950ba62deacf5c5faca89922f87972e11fc594cdc2ae1d0a5d02de::CHILLGUY::CHILLGUY",
      "0xabf27cc0cba747b355f4ed64e9263ddf1e6b38d2caf570df5b5777f722fa861e::BULLY::BULLY",

File: src/constants.tsx [hunk 2]
```diff
@@ -109,3 +112,339 @@ export const objectCoreResource = "0x1::object::ObjectCore";
...
+export const labsVerifiedTokens: Record<string, string> = {
+  "0x357b0b74bc833e95a115ad22604854d6b0fca151cecd94111770e5d6ffc9dc2b": "USDt",
+  "0xbae207659db88bea0cbead6da0ed00aac12edcdda169e591cd41c94180b46f3b": "USDC",
+  "0xd39fcd33aedfd436a1bbb576a48d5c7c0ac317c9a9bb7d53ae9ffb41e8cb9fd9":
+    "Find Out Points",
+  "0x50038be55be5b964cfa32cf128b5cf05f123959f286b4cc02b86cafd48945f89": "BUIDL",
+};
+export const MARKED_AS_SCAM = "Marked as scam";
+export const MARKED_AS_POSSIBLE_SCAM = "Marked as possible scam";
+export const labsBannedTokens: Record<string, string> = {
```
Comment (bv-aptos) [inline src/constants.tsx:429]: Should have left the comment here:

"0xf658475dc67a4d48295dbcea6de1dc3c9af64c1c80d4161284df369be941dafb::moon_coin::MoonCoin",
      "0xbbc4a9af0e7fa8885bda5db08028e7b882f2c2bba1e0fedbad1d8316f73f8b2f::Questss::Quests",
      "0xbbc4a9af0e7fa8885bda5db08028e7b882f2c2bba1e0fedbad1d8316f73f8b2f::quest::Quest",
      "0xbc106d0fef7e5ce159423a1a9312e011bca7fb57f961146a2f88003a779b25c2::QUEST::QUEST",
      "0xbbc4a9af0e7fa8885bda5db08028e7b882f2c2bba1e0fedbad1d8316f73f8b2f::Questseess::Questsee",
      "0xbbc4a9af0e7fa8885bda5db08028e7b882f2c2bba1e0fedbad1d8316f73f8b2f::graffio::GRAFFIO",
      "0xbbc4a9af0e7fa8885bda5db08028e7b882f2c2bba1e0fedbad1d8316f73f8b2f::ograffioo::Ograffioo",
      "0xbbc4a9af0e7fa8885bda5db08028e7b882f2c2bba1e0fedbad1d8316f73f8b2f::ograffio::Ograffio",
      "0xbbc4a9af0e7fa8885bda5db08028e7b882f2c2bba1e0fedbad1d8316f73f8b2f::memes::Memecoins",
      "0xbbc4a9af0e7fa8885bda5db08028e7b882f2c2bba1e0fedbad1d8316f73f8b2f::meme::Memecoin",
      "0xbbc4a9af0e7fa8885bda5db08028e7b882f2c2bba1e0fedbad1d8316f73f8b2f::tortugafin::Tortugafin",
      "0x397071c01929cc6672a17f130bd62b1bce224309029837ce4f18214cc83ce2a7::USDC::USDC",
      "0xbbc4a9af0e7fa8885bda5db08028e7b882f2c2bba1e0fedbad1d8316f73f8b2f::Ogrf::Ogrff",
      "0x24a808cbdeee2e8b771fec7ce9140bab2cbbf89426bf135d6b5d88e6736bb038::EZMoney::EZ_Money",
      "0x48327a479bf5c5d2e36d5e9846362cff2d99e0e27ff92859fc247893fded3fbd::APTOS::APTOS",
      "0xbbc4a9af0e7fa8885bda5db08028e7b882f2c2bba1e0fedbad1d8316f73f8b2f::Memecoins::Aptmeme",
      "0xbde343d6d2a7b407b1314c6c6ef771ebd72b95f9f67b688bbaea5ece4b98bb57::maincoin::Aptmeme",
      "0xbe5e8fa9dd45e010cadba1992409a0fc488ca81f386d636ba38d12641ef91136::maincoin::Aptmeme",
      "0xe1d39a72bd69bc2ebfe008bb925badb23a32883b077218b9e167f74cf703db1a::uptos::UptosCoin",
      "0x50788befc1107c0cc4473848a92e5c783c635866ce3c98de71d2eeb7d2a34f85::aptos_coin::AptosCoin",
      "0x50788befc1107c0cc4473848a92e5c783c635866ce3c98de71d2eeb7d2a34f85::apt_rewards::APTRewards",
      "0x50788befc1107c0cc4473848a92e5c783c635866ce3c98de71d2eeb7d2a34f85::aptos_coin::AptosCoin",
      "0xb8aed93b62950ba62deacf5c5faca89922f87972e11fc594cdc2ae1d0a5d02de::CHILLGUY::CHILLGUY",
      "0xabf27cc0cba747b355f4ed64e9263ddf1e6b38d2caf570df5b5777f722fa861e::BULLY::BULLY",

**Review by bv-aptos — APPROVED** [review]
Refactor looks good. Left more known scam coins to add. 

Approving with hope that you will add those before merging.

File: src/components/Table/VerifiedCell.tsx (hunk 1 above)
Comment (gregnazario) [reply src/components/Table/VerifiedCell.tsx:77]: let's do this in a separate pr

**Pull request merged**